packages/cli/src/cmds/account.ts

```diff
@@ -169,14 +181,37 @@ class AccountLogin extends YargsCommand {
         description: "Authenticate with a specific Azure Active Directory tenant.",
         type: "string",
         default: "",
+      })
+      .options("service-principal", {
```

Comment:
Anyway to show these options only for **azure**?

Reply:
Seperate login command to m365 and azure.